packages/nx/src/command-line/release/config/config.spec.ts

```diff
@@ -3313,6 +3331,7 @@ describe('createNxReleaseConfig()', () => {
                   "conventionalCommits": false,
                   "generator": "@nx/js:release-version",
                   "generatorOptions": {},
+                  "groupPreVersionCommand": "nx run-many -t build",
```

Comment:
This confirms what I said in my previous comment - this should not be set in this example, we do not want the command to be duplicated from global pre to group pre